deployment/ansible/roles/ckan-odp-configuration/defaults/main.yml

```diff
@@ -7,4 +7,5 @@ ckan_smtp_starttls: "False"
 ckan_smtp_password: ""
 ckan_database_backups: "/root/db_backups"
 ckan_database_backup_script: "/root/backup_db.sh"
+ckan_licenses: "/usr/local/share/licenses.json"
```

Comment:
This gets installed via ansible in the tasks/main.yml of this role.

https://github.com/azavea/opendataphilly-ckan/pull/30/files#diff-2db750f5cd207d122f8ee190c5afcadaR40